### ISSUE TITLE
Broken builds - GraphQL 4 - modelConfig

### ISSUE THREAD

**Issue by emteknetnz** [silverstripe/silverstripe-admin#1215]
https://travis-ci.com/github/silverstripe/silverstripe-admin/jobs/505604067 (admin suite)
https://travis-ci.com/github/silverstripe/silverstripe-admin/jobs/505604068 (cms suite, same issue)

We've recently started building the graphql 4 schema at the start of a behat test - https://github.com/silverstripe/silverstripe-graphql/pull/380

This seems to have exposed an issue within admin

```
In Schema.php line 1055:
  Config contains invalid keys: modelConfig  
```

### PRs
- [x] https://github.com/silverstripe/silverstripe-admin/pull/1224
- [x] https://github.com/silverstripe/silverstripe-admin/pull/1225

**Comment by emteknetnz** [comment]
Looks like it's frameworktest - using 0.4.0 instead of the more recent 0.4.2 -- needs this commit https://github.com/silverstripe/silverstripe-frameworktest/commit/3c83a619888a95edc93a0f6a095f028fd87cb1c3

**Comment by maxime-rainville** [comment]
All the PRs have been merged ... I've merged up 1.8 into 1. Just waiting to confirm that's green.

**Comment by maxime-rainville** [comment]
Travis is happy
